### PR TITLE
Regenerate track 1 crates

### PR DESCRIPTION
**Pull request by heaths**
- **Update codegen branch URL, versions**
- **Update authored branch URLs, versions**
- **Bump crate versions and ignore problematic api-version**
- **Regenerate mgmt, svc crates**

### CODE REVIEW

**Review by LarryOsterman — APPROVED** [review]

**Review by LarryOsterman — APPROVED** [review]

**Comment by LarryOsterman** [comment]
FWIW, the new PRs reference the legacy branch, which has not yet been created. I'm assuming that isn't going to be a problem.

**Review by LarryOsterman — APPROVED** [review]

File: services/mgmt/activedirectory/Cargo.toml [hunk 1]
```diff
@@ -46,5 +46,4 @@ default_tag = ["package-2020-03"]
 "package-2017-04-01" = []
 
 [lints]
-workspace = true
-
+workspace = true
```
Comment (LarryOsterman) [inline services/mgmt/activedirectory/Cargo.toml:49]: Missing cr at end of file?

Comment (heaths) [reply services/mgmt/activedirectory/Cargo.toml:49]: Odd. The template has a separate line break but the process of writing the file must've removed it. It's possible this file was modified manually at some point to fix a build break. I'm not going to fret about it, though.

**Comment by heaths** [comment]
If @LarryOsterman isn't good enough, I don't know who is. Overriding policy...

**Pull request merged**